src/analyses/flow_insensitive_analysis.cpp

```diff
@@ -217,7 +217,6 @@ bool flow_insensitive_analysis_baset::do_function_call(
     r->location_number=0;
 
     goto_programt::targett t=temp.add_instruction(END_FUNCTION);
-    t->code.set(ID_identifier, code.function());
```

Comment:
I think this might be lost from a different commit.  I'm not sure it is wrong but, thought you might want to know.